plugins/point.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: point
+spec:
+  version: "v0.1.0"
+  homepage: https://github.com/davidchua/kubectl-point
+  shortDescription: "Creates a headless service to an external source with optional ingress"
+  description: |
+    Generates and creates a headless service and/or ingress to point to any external source
```

Comment:
Add a trailing period and maybe go into just a little bit more into detail as to what the feature list is etc.